src/main/kotlin/gg/skytils/skytilsmod/listeners/DungeonListener.kt

```diff
@@ -150,7 +151,7 @@ object DungeonListener {
                             if (room.foundSecrets != sec) {
                                 room.foundSecrets = sec
                                 if (team.size > 1)
-                                    WSClient.sendPacketAsync(C2SPacketDungeonRoomSecret(SBInfo.server ?: return@setFoundSecrets, room.mainRoom.data.name, sec))
+                                    WSClient.sendPacketAsync(C2SPacketDungeonRoomSecret(SBInfo.server ?: return@setFoundSecrets, room.mainRoom.data.name, sec, mc.thePlayer.name))
```

Comment:
what is the point of this? two players can have the same update at the same time and can be in the same room at the same time

additionally the server is aware of the sender of the packet